CreeDictionary/CreeDictionary/templates/CreeDictionary/word-entries.html

```diff
@@ -1,19 +1,15 @@
 {% load inflection_extras %}
 {% load static %}
 
-{% for result in results %}
+{% for result in search_results %}
     <li class="search-results__result" data-cy="search-results">
         <article class="definition box box--rounded" data-cy="search-result">
             <header class="definition__header">
                 <h2 class="definition-title definition-title--search-result">
 
                     <dfn class="definition-title__word" data-cy="definition-title">
-                        {# upon loaded, this <a> will be attached to onclick handler that loads #}
-                        {# detailed lemma page. <a> is used for accessible styling #}
```

Comment:
now the url comes from url-reversing on the server side

Reply:
beauty!